prow.go

```diff
@@ -1082,6 +1082,10 @@ chmod ug+x /tmp/bin/oc
 oc policy add-role-to-group system:image-puller -n $(NAMESPACE) system:authenticated
 `
 
+const launchClusterScript = `ci-operator $@ &
```

Comment:
why isn't this just inlined?  Really no value in splitting it into its own string.

Reply:
At the time I wrote it, I was worried that I might need more logic than I ended up with.  I've in-lined the logic now.